kie-spring-boot/kie-spring-boot-samples/kie-server-spring-boot-kafka-sample/src/test/java/org/kie/server/springboot/samples/KafkaFixture.java

```diff
@@ -114,6 +115,12 @@
     
     protected KModuleDeploymentUnit unit = null;
     
+    public static void checkRightOSForTestContainers() {
+        // Currently testcontainers are not supported out-of-the-box on Windows and RHEL8
+        assumeTrue(!System.getProperty("os.name").toLowerCase().contains("windows") 
+                && !System.getProperty("os.version").contains("el8"));
```

Comment:
maybe as a small improvement we can also use `toLowerCase` so both conditions are mostly the same - but not really needed, 
```suggestion
                && !System.getProperty("os.version").toLowerCase().contains("el8"));
```

Reply:
Please take a look at https://mkyong.com/java/how-to-detect-os-in-java-systemgetpropertyosname/
I think is better to use win rather than windows